modules/stanford_course_views/stanford_course_views.module

```diff
@@ -5,3 +5,27 @@
  */
 
 include_once 'stanford_course_views.features.inc';
+
+function stanford_course_views_views_pre_build(&$view) {
+  // We want to provide a link to the search view when there are no results
+
+  if (($view->name == "stanford_courses_default") || ($view->name == "courses")){
```

Comment:
stanford_courses_default should no longer exist. The two view machine names are:

courses
courses_reference

Please check that the display names are consistent between the two.